lib/fastlane/actions/cordova_get_config_value.rb

```diff
@@ -0,0 +1,91 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      CORDOVA_GET_CONFIG_VALUE_CUSTOM_VALUE = :CORDOVA_GET_CONFIG_VALUE_CUSTOM_VALUE
+    end
+
+    class CordovaGetConfigValueAction < Action
+      def self.run(params)
+        Actions.verify_gem!('xml-simple')
+        require 'xmlsimple'
+        begin
+          config_file = File.open(File.expand_path(params[:path]))
+          config = XmlSimple.xml_in(config_file)
+
+          value = config[params[:key]]
+          Actions.lane_context[SharedValues::CORDOVA_GET_CONFIG_VALUE_CUSTOM_VALUE] = value
+
+          return value
+        rescue => ex
+          Helper.log.error ex
+          Helper.log.error "Unable to find config file at '#{path}'".red
```

Comment:
That's not true. Also, please remove the `rescue` block as this should fail and raise an exception if something goes wrong.